site/components/src/main/java/uk/nhs/digital/common/feed/RssModifier.java

```diff
@@ -261,11 +256,32 @@ public void modifyEntry(final HstRequestContext context, final Item entry, final
     @NotNull
     private Element getImageElement(String imageUrl) {
         final Element element
-            = new Element("thumbnail", "media", "http://search.yahoo.com/mrss/");
+            = new Element("content", "media", "http://search.yahoo.com/mrss/");
```

Comment:
Suggest you put this URL in a property file or resource bundle rather than hardcode it in this class.

Reply:
Can't be done atm